utilities/BuildUtilities.groovy

```diff
@@ -18,6 +18,8 @@ import groovy.ant.*
 @Field BuildProperties props = BuildProperties.getInstance()
 @Field HashSet<String> copiedFileCache = new HashSet<String>()
 @Field def gitUtils = loadScript(new File("GitUtilities.groovy"))
+@Field def depScannerUtils= loadScript(new File("DependencyScannerUtilities.groovy"))
```

Comment:
Just like other changes requested, could it be named dependencyScannerUtils instead of depScannerUtils, just for readability purpose?

Reply:
@M-DLB please have a look to the recent commit